### PR TITLE
Change equipment gear in trekking, move order, fix api v2 translate e…

### PR DESCRIPTION
**Pull request by LePetitTim**
…quipment

### CODE REVIEW

File: geotrek/api/v2/serializers.py [hunk 1]
```diff
@@ -1056,6 +1060,7 @@ class CourseSerializer(PDFSerializerMixin, DynamicFieldsMixin, serializers.Model
         parents = serializers.ReadOnlyField(source='parents_id')
         accessibility = serializers.SerializerMethodField()
         attachments = AttachmentSerializer(many=True, source='sorted_attachments')
+        equipment = serializers.SerializerMethodField()
```
Comment (submarcos) [inline geotrek/api/v2/serializers.py:1063]: I don't understand here why you delete equipment in fields but add attribute

File: geotrek/api/v2/serializers.py [hunk 2]
```diff
@@ -1069,6 +1074,9 @@ def get_accessibility(self, obj):
         def get_cities(self, obj):
             return [city.code for city in obj.published_cities]
 
+        def get_equipment(self, obj):
+            return get_translation_or_dict('equipment', self, obj)
```
Comment (submarcos) [inline geotrek/api/v2/serializers.py:1078]: idem

File: geotrek/api/v2/serializers.py (hunk 1 above)
Comment (LePetitTim) [reply geotrek/api/v2/serializers.py:1063]: This is not for the same model it is for outdoor course (we should translate it)

File: geotrek/api/v2/serializers.py (hunk 2 above)
Comment (LePetitTim) [reply geotrek/api/v2/serializers.py:1078]: This is not for the same model it is for outdoor course (we should translate it)

Comment (LePetitTim) [reply geotrek/api/v2/serializers.py:1078]: - Fix translation equipment api v2 courses

**Pull request merged**